### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gabehaack**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,10 @@
 FROM openjdk:11-jre
 
-ENV SCALA_VERSIONS='"2.12.10", "2.13.2"' \
-    SBT_VERSION=1.3.12 \
-    DOCKER_VERSION=18.09.9 \
-    AWS_CLI_VERSION=1.17.9 \
-    FLYWAY_VERSION=6.3.3 \
-    SERVERLESS_VERSION=1.67.3 \
+ENV SCALA_VERSIONS='"2.12.12"' \
```
Comment (gabehaack) [inline Dockerfile:3]: Want to target this scala version for some performance updates

File: Dockerfile [hunk 2]
```diff
@@ -15,11 +14,12 @@ WORKDIR /tmp
 # install baseline system packages
 RUN apt-get update \
     && apt-get install -y \
-    bash ca-certificates curl git jq openssh-client openssl python sudo tar ncurses-base
+    bash ca-certificates curl git groff jq openssh-client openssl python sudo tar ncurses-base
```
Comment (gabehaack) [inline Dockerfile:18]: groff is required by aws-cli v2

File: Dockerfile [hunk 3]
```diff
@@ -15,11 +14,12 @@ WORKDIR /tmp
...
 
-# install serverless framework
-RUN apt-get install -y nodejs npm \
-    && npm install -g serverless@${SERVERLESS_VERSION}
+# install node and serverless framework
+RUN curl -sL "https://deb.nodesource.com/setup_${NODE_VERSION}.x" | bash - \
+    && apt-get install -y nodejs \
+    && sudo npm install -g serverless
```
Comment (gabehaack) [inline Dockerfile]: Getting the latest node version

File: Dockerfile [hunk 4]
```diff
@@ -36,16 +36,16 @@ RUN apt-get install -y postgresql-11 \
 # install flyway
 RUN mkdir -p ${BUILD_PATH}/flyway \
     && cd ${BUILD_PATH}/flyway \
-    && curl -Ls -o flyway.tar.gz https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/${FLYWAY_VERSION}/flyway-commandline-${FLYWAY_VERSION}.tar.gz \
+    && curl -Ls -o flyway.tar.gz "https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/${FLYWAY_VERSION}/flyway-commandline-${FLYWAY_VERSION}.tar.gz" \
```
Comment (gabehaack) [inline Dockerfile:40]: Quotes make the variable highlight correctly in VSCode

File: Dockerfile [hunk 5]
```diff
@@ -36,16 +36,16 @@ RUN apt-get install -y postgresql-11 \
...
     && tar -xzf flyway.tar.gz --strip-components=1 \
     && rm flyway.tar.gz \
     && ln -s ${BUILD_PATH}/flyway/flyway /usr/local/bin/flyway \
     && cd -
 
-RUN curl -s -o awscli-bundle.zip "https://s3.amazonaws.com/aws-cli/awscli-bundle-${AWS_CLI_VERSION}.zip" \
-    && unzip awscli-bundle.zip \
-    && chmod +x ./awscli-bundle/install \
-    && ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
+# install aws-cli
```
Comment (gabehaack) [inline Dockerfile:46]: Use aws-cli v2

File: Dockerfile (hunk 3 above)
Comment (kylerisse) [inline Dockerfile]: it's nice to be able to hard core serverless when there are breaking changes. this happened to us on firewall.

**Review by kylerisse — APPROVED** [review]

File: Dockerfile (hunk 4 above)
Comment (kylerisse) [reply Dockerfile:40]: haha, plus it's the responsible way to interpolate strings!

File: Dockerfile (hunk 3 above)
Comment (gabehaack) [reply Dockerfile]: I have added the version we're on here - not going to push another image but it'll be in the repo for reference

**Pull request merged**